extensions/wrapper/wrapper/src/test/java/de/sovity/edc/ext/wrapper/api/ui/pages/policy/PolicyDefinitionApiServiceTest.java

```diff
@@ -81,39 +92,53 @@ void getPolicyList() {
     }
 
     @Test
-    void test_sorting(PolicyDefinitionService policyDefinitionService) {
+    void sortPoliciesFromNewestToOldest(DSLContext dsl) {
```

Comment:
These things would inject a DslContextFactory to force you to think of transactions.

If you had a DslContext injection here, it should be wrapped in a large transaction, which only works for tests that do not touch async edc processes